assets/js/util/errors.test.js

```diff
@@ -115,10 +116,37 @@ describe( 'Error Utilities', () => {
 		} );
 	} );
 
+	describe( 'isAuthError', () => {
+		it( 'should return TRUE if the error object has the `reconnectURL` property', () => {
+			const error = {
+				data: {
+					reconnectURL: 'example.com',
+				},
+			};
+
+			expect( isAuthError( error ) ).toBe( true );
+		} );
+
+		it( 'should return FALSE if the error object does not have `reconnectURL` property', () => {
+			const error = {
+				data: {
+					reason: 'dailyLimitExceeded',
+				},
+			};
+
+			expect( isAuthError( error ) ).toBe( false );
+		} );
+
+		it( 'should return FALSE if the passed object does not have the code property', () => {
+			expect( isAuthError( { message: 'Not Found' } ) ).toBe( false );
+		} );
```

Comment:
Let's make the declaration of the error inline with the other tests here so it's easier to parse:

```suggestion
		it( 'should return FALSE if the passed object does not have the `data` property', () => {
			const error = {
				message: 'Not Found',
			};

			expect( isAuthError( error ) ).toBe( false );
		} );
```